tsconfig.json

```diff
@@ -17,8 +18,10 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noEmit": false,
+    "noFallthroughCasesInSwitch": true,
+    "noUncheckedIndexedAccess": true,
```

Comment:
Also didn't have the habit of turning this on but upon reviewing it seems like a great value-add. I don't mind the extra boilerplate and sometimes find myself typing things w/ `| undefined` manually.

Reply:
I tend to be as strict as possible with the TS options